### PR TITLE
#456: Fix errors after user logs out

### PR DESCRIPTION
**Pull request by kkaczmarczyk**

### CODE REVIEW

File: src/main/webapp/app/shared/reducers/authentication.ts [hunk 1]
```diff
@@ -64,7 +70,8 @@ export default (state: AuthenticationState = initialState, action): Authenticati
     case ACTION_TYPES.LOGOUT:
       return {
         ...initialState,
-        showModalLogin: true
+        showModalLogin: true,
+        loggingOut: false
```
Comment (pmuchowski) [inline src/main/webapp/app/shared/reducers/authentication.ts]: I think you can also set it to false in the action above (login success) just to be safe

File: src/main/webapp/app/shared/reducers/authentication.ts [hunk 2]
```diff
@@ -32,6 +33,11 @@ export type AuthenticationState = Readonly<typeof initialState>;
 export default (state: AuthenticationState = initialState, action): AuthenticationState => {
   switch (action.type) {
     case REQUEST(ACTION_TYPES.LOGIN):
+    case REQUEST(ACTION_TYPES.LOGOUT):
+      return {
+        ...state,
+        loggingOut: true
```
Comment (pmuchowski) [inline src/main/webapp/app/shared/reducers/authentication.ts]: why it's set to true on login request?

Comment (kkaczmarczyk) [reply src/main/webapp/app/shared/reducers/authentication.ts]: fixeed

File: src/main/webapp/app/shared/reducers/authentication.ts (hunk 1 above)
Comment (kkaczmarczyk) [reply src/main/webapp/app/shared/reducers/authentication.ts]: ok

**Comment by kkaczmarczyk** [comment]
additionally I fixed case for LOGIN action success that seemed to be broken

**Pull request merged**